db/migrate/20190626132626_add_course_and_provider_ref_to_course_enrichment.rb

```diff
@@ -0,0 +1,34 @@
+class AddCourseAndProviderRefToCourseEnrichment < ActiveRecord::Migration[5.2]
+  def up
+    add_reference :course_enrichment, :course, index: true, type: :int
+    add_reference :course_enrichment, :provider, index: true, type: :int
```

Comment:
I don't think we want or need this ... it's just another place where something could go wrong/out of sync.

Reply:
yeah, i just blanket added it for convenience, will remove in due course